samples/order-processing-worker-nestjs/src/integration.spec.ts

```diff
@@ -0,0 +1,376 @@
+import { describe, expect, vi } from "vitest";
+import { Worker } from "@temporalio/worker";
+import { TypedClient } from "@temporal-contract/client";
+import { it as baseIt } from "@temporal-contract/testing/extension";
+import {
+  orderProcessingContract,
+  OrderSchema,
+} from "@temporal-contract/sample-order-processing-contract";
+import { extname } from "node:path";
+import { fileURLToPath } from "node:url";
+import type { z } from "zod";
+import { Client } from "@temporalio/client";
+import { Test, TestingModule } from "@nestjs/testing";
+import { ActivitiesProvider } from "./activities.provider.js";
+import { DependenciesModule } from "./dependencies.module.js";
+import { MockPaymentAdapter } from "./infrastructure/adapters/payment.adapter.js";
+
+type Order = z.infer<typeof OrderSchema>;
+
+const it = baseIt.extend<{
+  worker: Worker;
+  client: TypedClient<typeof orderProcessingContract>;
+  paymentAdapter: MockPaymentAdapter;
+  moduleRef: TestingModule;
+}>({
+  moduleRef: [
+    async (_context, use) => {
```

Comment:
```suggestion
    // oxlint-disable-next-line no-empty-pattern
    async ({}, use) => {
```

can you add this rule in AI guide lines